docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc

```diff
@@ -173,6 +173,25 @@ vault kv put clusters/kv/${TENANT_ID}/${CLUSTER_ID}/global-backup \
 # Generate a password for the cluster object backups
 vault kv put clusters/kv/${TENANT_ID}/${CLUSTER_ID}/cluster-backup \
   password=$(LC_ALL=C tr -cd "A-Za-z0-9" </dev/urandom | head -c 32)
+
+case "${VAULT_ADDR}" in
+  "https://vault-prod.syn.vshn.net")
+    export DAGOBERT_SOURCE="clusters/kv/t-falling-frog-2211/c-dawn-water-6350/dagobert"
+  ;;
+
+  "https://vault-int.syn.vshn.net")
+    export DAGOBERT_SOURCE="clusters/kv/t-ancient-morning-1764/c-old-fire-5788/dagobert"
+  ;;
+
+  *)
+    echo "Can not determine copy source for the Dagobert secrets" >&2
+    export DAGOBERT_SOURCE=""
+  ;;
+esac
+
+# Copy the Dagobert OpenShift Node Collector Credentials
+vault kv get -format=json "${DAGOBERT_SOURCE}" | jq '.data.data' \
+  | vault kv put -cas=0 "clusters/kv/${TENANT_ID}/${CLUSTER_ID}/dagobert" -
```

Comment:
Maybe better if we only try to copy dagobert credentials if we actually found a source

```suggestion
if [ -n "${DAGOBERT_SOURCE}" ]; then
vault kv get -format=json "${DAGOBERT_SOURCE}" | jq '.data.data' \
  | vault kv put -cas=0 "clusters/kv/${TENANT_ID}/${CLUSTER_ID}/dagobert" -
fi
```

Reply:
I plaed it nice and added even more output.